bin/MadGraph5_aMCatNLO/cards/production/13TeV/st_jj_rpv_LO/templates/PROCESSNAME_proc_card.dat

```diff
@@ -0,0 +1,16 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
+define p = g u c d s u~ c~ d~ s~ b b~
+define j = g u c d s u~ c~ d~ s~ b b~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```

Comment:
This is default, no? Please remove if so. You define q below but don't use it, or?

Reply:
Yes, I have removed the 
```
set group_subprocesses Auto
set ignore_six_quark_processes False
set loop_optimized_output True
set complex_mass_scheme False
```
And also removed the quark part.